Makefile

```diff
@@ -6,7 +6,7 @@ UNAME_S := $(shell uname -s)
 
 # If you're using docker and docker-compose, you can use this Makefile to run
 # commands in your docker images by setting the DOCKER_PREFIX environment variable
-# to: DOCKER_PREFIX="docker-compose run --rm web"
+# to: DOCKER_PREFIX="docker-compose run --rm web --"
```

Comment:
This actually needs to be fixed in `scripts/start-docker.sh`. I fixed it locally yesterday.